src/scripts/checksum_check.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Also... One experience from the past... While it is seemingly easier to write BASH scripts in simple cases, it's VERY quickly when the scripts become too complex/difficult to understand and maintain. I'd rather rewrite all those scripts in Python and use `uv` to run them in github actions with the nice support for standalone scripts that `uv` already has:

https://docs.astral.sh/uv/guides/scripts/#declaring-script-dependencies

This is PEP723  - https://peps.python.org/pep-0723/ that `uv` already supports and sooner or later it will be natively supported in Python environment (because that PEP is already approved). I think it's the right time to start using it in our automation.

Reply:
Lack of this "inline dependency declarations" was for me the one reason in the past I prefered to choose `bash` over Python, but since it is already there, and we have `uv` support for running such scripts and automatically creating and using the venv that is needed to do it, has removed the last obstacle for me where `python` scripts were not as good as `bash` scripts. With this one, I have absolutely no reason to use `bash` for scripting.

Reply:
(one thing it allows for example to add colors by declaring `rich` as dependency in such inline declaration).

Reply:
Yes sure, I like the python scripts to do these tasks. as this is initial draft and tried existing process in this. :). agree bash is bit complex to understand when it grows to bigger 😄